.env.sample

```diff
@@ -1,8 +1,10 @@
-NEXT_PUBLIC_BITCOIND_URL=
-NEXT_PUBLIC_EMILY_URL=
-NEXT_PUBLIC_MEMPOOL_API_URL=
-NEXT_PUBLIC_BITCOIN_RPC_USER_NAME=
-NEXT_PUBLIC_BITCOIN_RPC_PASSWORD=
-NEXT_PUBLIC_WALLET_NETWORK=
-NEXT_PUBLIC_SIGNER_AGGREGATE_KEY=
-NEXT_PUBLIC_CONTRACTS_DEPLOYER=
```

Comment:
`CONTRACTS_DEPLOYER` does not appear in the new config.

Reply:
will use SBTC_CONTRACT_ADDRESS instead

Reply:
At some point we should pivot to using `CONTRACTS_DEPLOYER` to fetch `SIGNER_AGGREGATE_KEY`, so having it in the config is kinda "preliminary work". Also with that information we can derive both the smart contract(s) and the token principal.

Reply:
Looking at `src/util/readOnlyHelper.ts`, it may be just a wording thing, but if so can we use something like `SBTC_DEPLOYER` for consistency with the signers config?

Reply:
created an issue for that @matteojug https://github.com/stacks-network/sbtc/issues/982